src/test/java/org/opensearch/geospatial/stats/upload/UploadStatsTests.java

```diff
@@ -5,12 +5,16 @@
 
 package org.opensearch.geospatial.stats.upload;
 
+import static org.hamcrest.collection.IsIn.in;
```

Comment:
Did you add this on purpose?

Reply:
It is supposed to be cleaned by spotApply :( I removed it now.